src/containers/ProjectProposals/ProjectProposalsRules/index.tsx

```diff
@@ -11,47 +11,87 @@ const ProjectProposalsRules: FC = () => {
         number={2}
-        text="Must be an open-source project."
+        text={
+          <>
+            Must be an <strong>open-source</strong> project.
+          </>
+        }
       />
       <StepIndicator
         className="ProjectProposalsRules__StepIndicator"
         number={3}
-        text="Direct distribution of thenewboston digital currency from the treasury is not allowed."
+        text={
+          <>
+            <strong>Direct distribution</strong> of thenewboston digital currency from the treasury is not allowed.
+          </>
+        }
       />
       <StepIndicator
         className="ProjectProposalsRules__StepIndicator"
         number={4}
-        text="Projects involving gambling or illegal activities of any kind are not allowed."
+        text={
+          <>
+            Projects involving <strong>gambling</strong> or <strong>illegal activities</strong> of any kind are not
+            allowed.
+          </>
+        }
       />
       <StepIndicator
         className="ProjectProposalsRules__StepIndicator"
         number={5}
-        text="Projects promoting abusive advertisements are not allowed."
+        text={
+          <>
+            Projects promoting <strong>abusive advertisements</strong> are <strong>not</strong> allowed.
+          </>
+        }
       />
       <StepIndicator
         className="ProjectProposalsRules__StepIndicator"
         number={6}
-        text="Projects dealing with explicit content or pornography are not allowed."
+        text={
+          <>
+            Projects dealing with <strong>explicit content</strong> or <strong>pornography</strong> are not allowed.
+          </>
+        }
       />
       <StepIndicator
         className="ProjectProposalsRules__StepIndicator"
         number={7}
-        text="Projects promoting pedophilia or inappropriate interaction targeted at a minor are not allowed."
+        text={
+          <>
+            Projects promoting <strong>pedophilia</strong> or <strong>inappropriate interaction</strong> targeted at a
+            minor are not allowed.{' '}
+          </>
+        }
       />
       <StepIndicator
         className="ProjectProposalsRules__StepIndicator"
         number={8}
-        text="Projects which use the user's personal information for any purpose must take prior permission from the user or inform them clearly."
+        text={
+          <>
+            Projects which use <strong>the user's personal information</strong> for any purpose must take prior
+            permission from the user or inform them clearly.{' '}
+          </>
+        }
       />
       <StepIndicator
         className="ProjectProposalsRules__StepIndicator"
         number={9}
-        text="thenewboston is not liable for copyright or other intellectual property rights, property damage, misuse of coins."
+        text={
+          <>
+            <strong>Thenewboston</strong> is not liable for <strong>copyright</strong> or other{' '}
```

Comment:
`thenewboston` instead of `Thenewboston`?

Reply:
Yes, last time I provided the doc to Bucky to update the site. It was supposed to be "Thenewboston" because it is the first word of the sentence. Also, he missed making some words bold which was given in the google docs. So In this PR, I am fixing them too.

Reply:
Thanks, @jamessspanggg for pointing out that. I think would be better to go with **thenewboston** with lowercase T.

Reply:
Thanks for pointing out that. I think would be better to go with **thenewboston** with lowercase **t**.